system-test/spanner.ts

```diff
@@ -1002,15 +999,7 @@ describe('Spanner', () => {
       if (IS_EMULATOR_ENABLED) {
         this.skip();
       }
-      database1 = instance.database(generateName('database'));
-      const [, operation] = await database1.create({
-        schema: `
-              CREATE TABLE Singers (
-                SingerId STRING(1024) NOT NULL,
-                Name STRING(1024),
-              ) PRIMARY KEY(SingerId)`,
-      });
-      await operation.promise();
+      database1 = DATABASE;
 
       await database1.table('Singers').insert({
```

Comment:
We should have a global `TABLE_NAME` defined so we don't have to assume this name exists.

Reply:
Thank you!